src/modules/NetworkController/private/Copy-CertificatesToFabric.ps1

```diff
@@ -38,7 +43,7 @@ function Copy-CertificatesToFabric {
             throw New-Object System.NullReferenceException("Unable to locate .pfx files under the specified CertPath location")
         }
 
-        "Retrieving Rest Certificate" | Trace-Output
+        "Retrieving Rest Certificate" | Trace-Output -Level:Verbose
         $currentRestCertificate = Get-SdnNetworkControllerRestCertificate
```

Comment:
Can this be simplified to provide $NcRestName as the parameter instead of self-detect?  Get-SdnNetworkControllerRestCertificate depends on Get-NetworkController will be failed in expired cert case. Or we update Get-SdnNetworkControllerRestCertificate to work with expired case. Reference Get-SdnNetworkControllerInfoOffline to read rest cert info from manifest.